ivy/container/general.py

```diff
@@ -2788,6 +2788,12 @@ def to_scalar(
             c: -1
         }
 
+        >>> print(ivy.is_int_dtype(y))
```

Comment:
This is not necessary 😅 Just showing the output of `to_scalar()` itself should be sufficient!

Thanks!